doc/man3/X509_check_purpose.pod

```diff
@@ -0,0 +1,54 @@
+=pod
+
+=head1 NAME
+
+X509_check_purpose - Checks certificate I<x's> purpose
+
+=head1 SYNOPSIS
+
+#include <openssl/x509v3.h>
+
+int X509_check_purpose(X509 *x, int id, int ca)
+
+=head1 DESCRIPTION
+
+This function checks if B<certificate> I<x> was created with the B<purpose>
+represented by I<id>. If I<ca> is non-zero, then certificate I<x> is
+checked to determine if it's a possible CA with various levels of certainty
+possibly returned.
+
+Below are the potential ID's that can be checked:
+# define X509_PURPOSE_SSL_CLIENT        1
+# define X509_PURPOSE_SSL_SERVER        2
+# define X509_PURPOSE_NS_SSL_SERVER     3
+# define X509_PURPOSE_SMIME_SIGN        4
+# define X509_PURPOSE_SMIME_ENCRYPT     5
+# define X509_PURPOSE_CRL_SIGN          6
+# define X509_PURPOSE_ANY               7
+# define X509_PURPOSE_OCSP_HELPER       8
+# define X509_PURPOSE_TIMESTAMP_SIGN    9
```

Comment:
Please separate the "Below..." line from the `# define` lines, and indent all the `# define` lines with one space.

The two things to remember is that PODs are treated a paragraph at a time, and paragraphs are separated with empty lines, and that lines indented with at least one space are treated as verbatim, including line breaks (which are otherwise ignored).

Reply:
Fixed.